### PR TITLE
bolt: Add package version 0.9.2

### PR DESCRIPTION
**Pull request by suresh-thelkar**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Add package bolt version 0.9.2
###### Change Log  

- NA

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- NA

###### Links to CVEs  
- NA

###### Test Methodology

Buddy build
- [AMD64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=287103&view=results)
- [ARM64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=287104&view=results)

### CODE REVIEW

File: SPECS-EXTENDED/bolt/bolt.spec [hunk 1]
```diff
@@ -0,0 +1,167 @@
+Summary:        Thunderbolt device manager
+Name:           bolt
+Version:        0.9.2
+Release:        2%{?dist}
+License:        LGPLv2+
```
Comment (mfrw) [inline SPECS-EXTENDED/bolt/bolt.spec:5]:
```suggestion
License:        LGPL-2.0-or-later
```

File: SPECS-EXTENDED/bolt/bolt.spec [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+BuildRequires:  systemd-devel
+BuildRequires:  pkgconfig(gio-2.0)
+%{?systemd_requires}
+# for the integration test (optional)
+%if 0%{?fedora} || 0%{?rhel} > 8
+BuildRequires:  pygobject3-devel
+BuildRequires:  python3-dbus
+BuildRequires:  python3-dbusmock
+BuildRequires:  umockdev-devel
+%endif
```
Comment (mfrw) [inline SPECS-EXTENDED/bolt/bolt.spec:26]: Can we double check to see if these deps make sense for us.

I think our preference is to not have `%if fedora/rhel` if possible.

File: SPECS-EXTENDED/bolt/bolt.spec (hunk 1 above)
Comment (suresh-thelkar) [reply SPECS-EXTENDED/bolt/bolt.spec]: Done

File: SPECS-EXTENDED/bolt/bolt.spec (hunk 2 above)
Comment (suresh-thelkar) [reply SPECS-EXTENDED/bolt/bolt.spec]: Thanks. I have removed the above block of code. Ran Buddy builds after that. Here are the results. 
- [AMD64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=305821&view=results)
- [ARM64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=305822&view=results)

**Comment by suresh-thelkar** [comment]
@mfrw, I have addressed your comments. Can you please take a look.

**Review by mfrw — APPROVED** [review]

**Comment by suresh-thelkar** [comment]
I have once again ran Buddy Builds after rebasing with mainline. Both of them are passed. Here are the results
- [AMD64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=313357)
- [ARM64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=313358)

**Pull request merged**